src/tmdbService.js

```diff
@@ -1,59 +1,59 @@
-import { useState, useEffect } from "react"
-import axios from "axios"
-import { apiConfig } from "./apiConfig.js"
+import { useState, useEffect } from "react";
```

Comment:
There are a lot of semicolons in this file that don't exist anywhere else. I would check your linter.